http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/ProxyRoutes.scala

```diff
@@ -237,17 +239,28 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport, refererC
     for {
       ctx <- ev.ask[AppContext]
       apiCall = proxyService.proxyRequest(userInfo, googleProject, runtimeName, request)
-      _ <- metrics.incrementCounter("proxyRuntime")
       resp <- ctx.span.fold(apiCall)(span => spanResource[IO](span, "proxyRuntime").use(_ => apiCall))
+
+      headerMap: Map[String, String] = request.headers.map(header => (header.name(), header.value())).toMap
+      contentLength = headerMap.get("content-length").toString.toDouble
+      _ <- if (request.uri.toString() contains ".ipynb") {
```

Comment:
Does it make sense to check if `request.uri` not only `contains` `.ipynb` but also ends with it?